### DISCUSSION TITLE
Question - How to use Skia with Mapsui

### DISCUSSION THREAD

**Discussion by BrianVance** [Mapsui/Mapsui#1193] [Q&A] [answered]
Hi,
   I need to draw an arc (green line) between two lines. I don't think Mapsui has this control by default and we would need to leverage SkiaSharp for this purpose. 

I have read an article on drawing an Arc but I am not sure how to call this from Mapsui. The map.Drawables.Add only works with Drawable items. Is there something specific for Skia? Any help would be appreciated.

https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/curves/arcs

![image](https://user-images.githubusercontent.com/10665687/132375961-12f4f623-5f8b-4c06-b869-bd2b7587599e.png)

Regards,
Brian Noronha

**Comment by charlenni** [comment, marked as answer]
You could us a LineString (Polyline) for this. If you use enough points, you get an arc. Look on the source code of Circle.cs of Mapsui.UI.Forms and the corresponding demo in the samples. There you could see, how circles could be created by lines.

**Reply by BrianVance** [reply]
Thanks this was very helpful
